cmd/bundle/run.go

```diff
@@ -19,7 +19,27 @@ func newRunCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "run [flags] KEY",
 		Short: "Run a resource (e.g. a job or a pipeline)",
-		Args:  root.MaximumNArgs(1),
+		Long: `Run a resource (e.g. a job or a pipeline)
+
+The KEY argument is the unique identifier of the resource to run. You can customize the run by
+specifying any of the flags below, or if applicable, additional positional arguments.
+
+Additional positional arguments can be specified like this:
+
+   databricks bundle run my_job -- --key1 value1 --key2 value2
+
+Or:
+
+   databricks bundle run my_job -- example positional arguments
```

Comment:
I guess these are argument values?  If so, maybe the example would be clearer as `databricks bundle run my_job -- value1 value2 value3`?